src/core/ddsi/src/ddsi_typewrap.c

```diff
@@ -454,11 +454,23 @@ static void xt_lbounds_dup (struct DDS_XTypes_LBoundSeq *dst, const struct DDS_X
   dst->_buffer = ddsrt_memdup (src->_buffer, dst->_length * sizeof (*dst->_buffer));
 }
 
+static void get_namehash (DDS_XTypes_NameHash name_hash, const char *name)
+{
+  /* FIXME: multi byte utf8 chars? */
```

Comment:
If one doesn't need to interpret them, then handling as simple sequences of bytes is ok, so I think we need not worry about that